helm_deploy/values-preprod.yaml

```diff
@@ -30,3 +30,15 @@ whitelist:
   cloudplatform-live1-1: "35.178.209.113/32"
   cloudplatform-live1-2: "3.8.51.207/32"
   cloudplatform-live1-3: "35.177.252.54/32"
+
+kiam:
+  roleArn: arn:aws:iam::754256621582:role/offender-search-elastic-search-delius-pre-prod
+
+elasticSearch:
+  deliusRoleArn: arn:aws:iam::010587221707:role/cp-offender-search-service-role-delius-pre-prod
+  host: vpc-newtech-search-2zhrbjdmz4jbqx4i42f4brgoji.eu-west-2.es.amazonaws.com
+  port: 443
+  scheme: https
+
+nomisOAuth:
+  JWT_JWK_SET_URI: https://gateway.t3.nomis-api.hmpps.dsd.io/auth/.well-known/jwks.json
```

Comment:
Looks like test url, let me find the correct one

Reply:
https://gateway.preprod.nomis-api.service.hmpps.dsd.io/auth/.well-known/jwks.json